Aprendizado por Reforço Clássico/Bandits/Agente Epsilon-Guloso/eps_greedy.ipynb

```diff
@@ -152,33 +159,33 @@
     "num_runs = 200\n",
     "num_steps = 1000\n",
     "np.random.seed(1)\n",
-    "total_medias = []\n",
+    "total_means = []\n",
     "\n",
-    "media_maxima = 0\n",
+    "max_mean = 0\n",
     "for runs in range(num_runs):\n",
     "\n",
-    "    agente = AgenteEpsilonGuloso(k_bracos=10, epsilon=0.1)\n",
+    "    agent = EpsilonGreedy(k_arms=10, epsilon=0.1)\n",
     "\n",
-    "    env = bandit.GaussianBandit(k_bracos=10)\n",
+    "    env = bandit.GaussianBandit(k_arms=10)\n",
     "\n",
-    "    pontuacao = [0]\n",
-    "    medias = []\n",
+    "    score = [0]\n",
+    "    means = []\n",
     "\n",
-    "    media_maxima += np.max(env.bandits_expectations)\n",
+    "    max_mean += np.max(env.bandits_expectations)\n",
     "\n",
     "    for steps in range(num_steps):\n",
-    "        recompensa = env.gamble(agente.ultima_acao)\n",
-    "        agente.ultima_acao = agente.agente_step(recompensa)\n",
+    "        reward = env.gamble(agent.last_action)\n",
+    "        agent.last_action = agent.agent_step(reward)\n",
     "\n",
-    "        pontuacao.append(pontuacao[-1] + recompensa)\n",
-    "        medias.append(pontuacao[-1]/(steps+1))\n",
-    "    total_medias.append(medias)\n",
+    "        score.append(score[-1] + reward)\n",
+    "        means.append(score[-1]/(steps+1))\n",
+    "    total_means.append(means)\n",
     "\n",
     "plt.figure(figsize=(15,5))\n",
-    "plt.plot([media_maxima / num_runs for _ in range(num_steps)], linestyle=\"--\")\n",
-    "plt.plot(np.mean(total_medias, axis=0))\n",
+    "plt.plot([max_mean / num_runs for _ in range(num_steps)], linestyle=\"--\")\n",
+    "plt.plot(np.mean(total_means, axis=0))\n",
     "plt.legend([\"Melhor Possível\", \"Epsilon-Guloso\"])\n",
-    "plt.title(\"Recompensa Média de um Agente Epsilon-Guloso\")\n",
+    "plt.title(\"Recompensa Média de um agent Epsilon-Guloso\")\n",
```

Comment:
Era pra ter mudado de agente pra agent mesmo? Ou foi acidental?

Reply:
totalmente acidental. Quando tava traduzindo, eu fiz um "find and replace" all nas variáveis e esqueci que ia pegar algumas strings.